### DISCUSSION TITLE
Segmentation fault when running lammps with deepmd + kspace + bond/angle information

### DISCUSSION THREAD

**Discussion by dkussainova** [deepmodeling/deepmd-kit#4184] [Q&A]
Hi,

I'm trying to run lammps with hybrid/scaled pair_style to combine deepmd model and empirical model. Therefore, I'm using atom_style full with data file that contains coordinates, bonds, and angles. To use lj/cut/coul/long I'm also trying to use kspace (kspace_style pppm 1e-5 kspace_modify gewald 0.45). However, I'm getting segmentation fault. After some tests, I identified that I can run successfully deepmd model with full atom style and data file that includes bond/angle information but without kspace. Or I can run deepmd with kspace and full atom style but bonds and angles are removed from the data file. But when I combine all of them I'm getting segmentation fault. Is there some reason for this? And can it be fixed? 

Any help will be greatly appreciated.

Thanks,
Dina

Here is an error message:
![error_message](https://github.com/user-attachments/assets/ab99a77c-d7b1-4d7f-bba7-ec641a823ec7)
I also tried to run with different lammps/deepmd, but getting similar error:
![error_message2](https://github.com/user-attachments/assets/e5680ae8-bbcb-4bbb-ba48-900192917bab)
Input lammps file:
![input_file](https://github.com/user-attachments/assets/eb189647-13bb-45f5-9afe-01ac80e2598b)
Data file for lammps:
![data_file](https://github.com/user-attachments/assets/27793f7d-be58-49dc-930e-1a8f6da1310a)

**Comment by njzjz** [comment]
Have you used the latest version?

**Reply by dkussainova** [reply]
I have tried to run it with Lammps Aug 2 2023 + DeePMD-kit v2.2.10 and also with Lammps Sep 29 2021 + Deepmd v2.1.5
